vowpalwabbit/vwdll.h

```diff
@@ -67,6 +67,7 @@ VW_DLL_MEMBER VW_HANDLE VW_CALLING_CONV VW_Initialize(const char16_t * pstrArgs)
 #endif
 VW_DLL_MEMBER VW_HANDLE VW_CALLING_CONV VW_InitializeA(const char * pstrArgs);
 VW_DLL_MEMBER VW_HANDLE VW_CALLING_CONV VW_InitializeWithModel(const char * pstrArgs, const char * modelData, size_t modelDataSize);
+VW_DLL_MEMBER VW_HANDLE VW_CALLING_CONV VW_SeedWithModel(VW_HANDLE handle, const char * extraArgs);
```

Comment:
@JohnLangford who is using this API?

Reply:
I assume @apoorvreddy ?  It's not yet in the master branch.

Reply:
I was asking about the C API in general, not just this function

Reply:
I don't believe that is answerable.

Reply:
@eisber My company is using VW in production systems. And we're using this API :)